plugin-dev/Source/Sentry/Private/Desktop/Transport/SentryTransport.cpp

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright (c) 2023 Sentry. All Rights Reserved.
+
+#include "SentryTransport.h"
+
+#include "SentryDefines.h"
+#include "SentryModule.h"
+#include "SentryDsnUrl.h"
+#include "SentrySettings.h"
+
+#include "Infrastructure/SentryConvertorsDesktop.h"
+
+#include "HttpModule.h"
+#include "HttpManager.h"
+#include "Misc/ScopeLock.h"
+
+#if USE_SENTRY_NATIVE
```

Comment:
Shouldn't that check for Linux? 
```suggestion
#if PLATFORM_LINUX & USE_SENTRY_NATIVE
```